providers/azure/resources/azure.lr.manifest.yaml

```diff
@@ -1470,6 +1483,10 @@ resources:
       properties: {}
       tags: {}
       type: {}
+      wafConfig:
```

Comment:
is this intention to have both?

Reply:
nope, likely a rename that snuck its way into the manifest, removed